src/native/external/LinuxTracepoints.cmake

```diff
@@ -0,0 +1,11 @@
+# IMPORTANT: do not use add_compile_options(), add_definitions() or similar functions here since it will leak to the including projects
+
+include_directories(${CMAKE_CURRENT_LIST_DIR}/LinuxTracepoints/libtracepoint/include)
```

Comment:
A nit - the `${CMAKE_CURRENT_LIST_DIR}/` should not be needed, cmake doc says:
> Relative paths are interpreted as relative to the current source directory.

Reply:
`CURRENT_LIST_DIR` is needed because the current "source" directory is the directory of the CMakeLists.txt that includes this file.

Reply:
Yeah, I originally tried without it and it failed to find the files